src/openrct2/Game.h

```diff
@@ -173,3 +173,4 @@ void rct2_to_utf8_self(char* buffer, size_t length);
 void game_fix_save_vars();
 void start_silent_record();
 bool stop_silent_record();
+void PrepareMapForSave();
```

Comment:
The function name seems a bit out of line with the others

Reply:
It is following the new rules, the other functions just haven’t been updated yet.